ui/tests/integration/components/kv/page/kv-page-overview-test.js

```diff
@@ -16,6 +16,8 @@ import { dateFromNow } from 'core/helpers/date-from-now';
 import { baseSetup } from 'vault/tests/helpers/kv/kv-run-commands';
 
 const { overviewCard } = GENERAL;
+
+// subkey is enterprise only but we don't have any version testing here because the @subkey arg is null for non-enterprise versions
```

Comment:
```suggestion
// subkeys access is enterprise only (in the GUI) but we don't have any version testing here because the @subkeys arg is null for non-enterprise versions
```